backend/saves.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+from dataclasses import dataclass
+from sqlite3 import connect
+import json
+
+@dataclass
+class Saves():
```

Comment:
Sweet, I think you proved idea of saving the rounds as JSON seems like a reasonable course of action. Good job!

I think we could easily move this into Starlette. Thinking about how we could expand what do you think about initially starting with two tables a `Game` and `Round` table, `Round` would essentially hold each round (ie state) of a game? This way we would have a full history of all the games, and we could just `ORDER BY modified DESC LIMIT 1` to always have the current state of the game?
<img width="481" alt="Screenshot 2023-01-09 at 9 09 52 AM" src="https://user-images.githubusercontent.com/7703829/211327295-a4487076-e150-4579-8318-be29d44a4a49.png">

Reply:
this is great. makes sense to me and I got something working this way. later, we might extend Game table to indicate final completed status too. If we decide to load old games from db on startup, I bet we'd exclude completed.